sql/testdata/grant_database

```diff
@@ -30,6 +24,18 @@ a        readwrite READ,WRITE
 a        root      READ,WRITE
 a        test-user READ,WRITE
 
+# Create table to inherit DB permissions.
+statement ok
+CREATE TABLE a.t (id INT PRIMARY KEY)
+
+query TTT
+SHOW GRANTS ON a.t
+----
+Table    User      Privileges
```

Comment:
FYI, you'll have to either remove this line or add the `colnames` option after rebasing on top of my update to the logic test framework.